modules/animationqt/src/animationqtsupplier.cpp

```diff
@@ -48,7 +48,12 @@ AnimationQtSupplier::AnimationQtSupplier(InviwoApplication* app)
                         IvwContextCustom("AnimationQtSupplier"));
     }()} {}
 
-AnimationQtSupplier::~AnimationQtSupplier() { unRegisterAll(); }
+AnimationQtSupplier::~AnimationQtSupplier() {
+    // Unregister everything from the factory. This is neccessary even though the base class
```

Comment:
This is as it should be no need for a comment here...

Reply:
fixed in d1540e2